assets/quick-order-form.css

```diff
@@ -20,4 +28,115 @@
 .quick-order-form .variant-item__price .price,
 .quick-order-form .variant-item__totals .price {
   font-size: 1.4rem;
-}
\ No newline at end of file
+}
+
+.quick-order-form a.variant-item__name {
+  text-decoration: none;
+}
+
+.quick-order-form a.variant-item__name:hover {
+  text-decoration: underline;
+  text-underline-offset: 0.2rem;
+  text-decoration-thickness: 0.1rem;
+}
+
+.quick-order-form .quick-order-list__table th {
+  border: 0;
+}
+
+.quick-order-form tr.variant-item:not(.product):hover,
+.quick-order-form tr.variant-item:not(.product):focus-within {
+  background-color: rgba(var(--color-foreground), 0.02);
+}
+
+.quick-order-form tr.variant-item td.variant-item__inner {
+  display: table-cell;
+}
+
+.quick-order-form .quick-order-list__table td {
+  padding-top: 0.4rem;
+  padding-bottom: 0.4rem;
+}
+
+.quick-order-form tr.product td {
+  min-height: calc((var(--inputs-border-width) * 2) + 5rem);
+  padding-top: 1.6rem;
+  padding-bottom: 1.6rem;
+}
+
+.quick-order-form .product-item__title--wrapper {
+  display: flex;
+}
+
+.quick-order-form .product {
+  box-shadow: inset 0px -0.1rem 0px 0px rgba(var(--color-foreground), 0.08);
+}
+
+.quick-order-form tr .variant-item__image-container {
+  margin-right: 0.8rem;
+}
+
+.quick-order-form tr.product .variant-item__image-container {
+  width: 3.6rem;
+  border: min(var(--media-border-width), 1.8rem) solid rgba(var(--color-foreground),var(--media-border-opacity));
+  height: auto;
+}
+
+.quick-order-form tr:not(.product) .variant-item__image-container {
+  width: 2.6rem;
+  height: 2.6rem;
+  border: min(var(--media-border-width), 1.3rem) solid rgba(var(--color-foreground),var(--media-border-opacity));
+}
+
+.quick-order-form .variant-item__image-container--no-img {
+  display: none;
+}
+
+.quick-order-form tr .variant-item__media {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-right: 0.8rem;
+}
+
+.quick-order-form tr:not(.product) .variant-item__media {
+  width: 2.6rem;
+  height: 4rem;
+}
+
+.quick-order-form tr.product .variant-item__media {
+  width: 3.6rem;
+  height: auto;
+}
+
+.quick-order-form .field {
+  margin-bottom: 1.8rem;
+}
+
+.quick-order-form .field label {
+  opacity: 0.3;
+}
+
+.search--wrapper {
```

Comment:
If this is specific only to quick order form, let's call it something like `quick-order-form-search--wrapper` to avoid confusion with the existing search

Reply:
that makes sense, thanks @sofiamatulis